charts/spire/charts/spire-agent/README.md

```diff
@@ -109,3 +109,6 @@ A Helm chart to install the SPIRE agent.
 | `experimental.enabled`                            | Allow configuration of experimental features                                                                                                                                                      | `false`                                                                          |
 | `experimental.syncInterval`                       | Sync interval with SPIRE server with exponential backoff                                                                                                                                          | `5s`                                                                             |
 | `experimental.featureFlags`                       | List of developer feature flags                                                                                                                                                                   | `[]`                                                                             |
+| `sockets.hostBasePath`                            | Path on which the agent socket is made available when admin.expose is true                                                                                                                        | `/run/spire/agent/sockets`                                                       |
```

Comment:
This set of values is leaving room in the future to fix the agent api socketPath option to also reuse sockets.hostBasePath for easier configuration. I didn't do that refactor here though as its a lot more changes then adding a new socket.

Reply:
Perhaps the future fix can include the renaming across all elements?  The commit as-is will have a confusing setup where some elements are configured via `socket` and others `sockets`.  I'd feel more comfortable putting the "future fix" stuff in a different commit that solely addresses the goals of the future fix, or expanding the scope to fully encompass the future fix such that all of it is delivered in this PR.

Reply:
What do you think of moving this config under sockets.admin so its clear it applies only to the admin socket? Also need to change admin.expose -> admin.mountOnHost

Reply:
I have mixed feelings.  If we get into the habit of accepting work in the wrong contexts, I worry that perhaps we will find it hard to roll back to denying work that is "directionally correct" in issues that have nothing to do with the item under review.

Under the worst cases, this means that we can't review with a scope in mind, because only some of the work will be present in the issue that eventually has the stage when the final bits of work are completed.  In this project and others, we've found bugs in the "pre submitted" work that wasn't caught in review under the final work, because the final work didn't have the controls of indicating all the changed files tied to the issue.

And the "ends justify the means" handling of it here rubs me wrong.

That said, I did say I'd align to your wishes, and in the end, if it's how you work, I'm not going to alter it.

Approved